public/index.html

```diff
@@ -5,6 +5,8 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js'></script>
```

Comment:
Are you sure that you need the mapbox-gl-js file from the CDN? If you search (cmd-f) your package-lock.json, you'll find that dependency is already being loaded from there

Reply:
thanks, there's warning asking to load mapbox css. I think your approach makes it better